docs/content/2.deploy/providers/vercel.md

```diff
@@ -35,6 +35,58 @@ It is possible to deploy your nitro applications directly on [Vercel Edge Functi
 
 In order to enable this target, please set `NITRO_PRESET` environment variable to `vercel-edge`.
 
+## Vercel Storage
+
+Nitro has built-in support for [Vercel KV storage](https://vercel.com/docs/storage/vercel-kv) and can be integrated with [Storage/Layer](/guide/storage).
```

Comment:
```suggestion
Nitro has built-in support for [Vercel KV storage](https://vercel.com/docs/storage/vercel-kv) which can be integrated with [storage/layer](/guide/storage).
```

Reply:
I have rephrased this title. can you please check latest?

Reply:
perfect 👌